pkg/controller/migcluster/migcluster_controller.go

```diff
@@ -116,7 +116,7 @@ type ReconcileMigCluster struct {
 // +kubebuilder:rbac:groups=migration.openshift.io,resources=migclusters,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=migration.openshift.io,resources=migclusters/status,verbs=get;update;patch
 func (r *ReconcileMigCluster) Reconcile(request reconcile.Request) (reconcile.Result, error) {
-	log.Info("Reconcile", "request", request.Name)
+	log = logf.Log.WithName(names.SimpleNameGenerator.GenerateName("cluster|"))
```

Comment:
Ah cool! I was thinking there was probably a good way to do this... Props on finding something in k8s.